public_html/wp-content/plugins/wordcamp-speaker-feedback/includes/admin.php

```diff
@@ -478,6 +482,95 @@ function handle_bulk_edit_actions( $action, $nonce ) {
 	return $messages;
 }
 
+/**
+ * Add extra actions to the list of links displayed for each comment.
+ *
+ * @param array      $actions An array of comment actions.
+ * @param WP_Comment $comment Comment data object.
+ *
+ * @return array
+ */
+function feedback_extra_actions( $actions, $comment ) {
+	$comment_id = $comment->comment_ID;
+	$feedback = get_feedback_comment( $comment );
+	if ( is_null( $feedback ) ) {
+		return $actions;
+	}
+
+	$unapprove_url = add_query_arg(
+		array(
+			'bulk_edit[]' => $comment_id,
+			'action' => 'unmark-inappropriate',
+			'_wpnonce' => wp_create_nonce( "inappropriate-comment_{ $comment_id }" ),
+			'bulk_edit_nonce' => wp_create_nonce( 'bulk_edit_' . COMMENT_TYPE ),
+		),
+		get_subpage_url( get_post_type( $comment->comment_post_ID ) )
+	);
+
+	$inappropriate_url = add_query_arg(
+		array(
+			'bulk_edit[]' => $comment_id,
+			'action' => 'mark-inappropriate',
+			'_wpnonce' => wp_create_nonce( "inappropriate-comment_{ $comment_id }" ),
+			'bulk_edit_nonce' => wp_create_nonce( 'bulk_edit_' . COMMENT_TYPE ),
+		),
+		get_subpage_url( get_post_type( $comment->comment_post_ID ) )
+	);
+
+	if ( 'inappropriate' === $comment->comment_approved ) {
+		$actions['unapprove'] = sprintf(
+			'<a href="%1$s" data-wp-lists="%2$s" class="vim-u aria-button-if-js" aria-label="%3$s">%4$s</a>',
+			esc_url( $unapprove_url ),
+			// This is meta information for wpList, to trigger an action when clicked. Most of it can be ignored,
+			// the relevant part is the last: query params to pass to admin-ajax.php. See wp_ajax_mark_inappropriate
+			// for how these are used.
+			// See https://core.trac.wordpress.org/browser/trunk/src/js/_enqueues/lib/lists.js?rev=46800#L217.
+			"delete:the-comment-list:comment-{$comment_id}:e7e7d3:action=dim-comment&amp;new=uninappropriate",
+			esc_attr__( 'Move this comment to pending', 'wordcamporg' ),
+			__( 'Pending', 'wordcamporg' )
```

Comment:
While the `Inappropriate` label seems pretty self-explanatory and doesn't need a verb, I think it would be a bit more intuitive on this one if the label was `Move to Pending` or something, so that it's clear where it's going.

Reply:
Sure, that works— I was thinking about "Spam" and "Trash" as nouns, so I missed that they're all actually verbs, ha. Will change it.

Reply:
Changed to  "Move to Pending" in 17f0c0d9443308ea58662bfe1bcb331fda93f062